src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -1,5 +1,5 @@
 <div>
-  <h2 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes(true)">Sessions</h2>
+  <h1 class="color-blue" *ngIf="showQuestion.length === 0 || showQuestion.includes(true)">Sessions</h1>
```

Comment:
The update of font sizes should be across the `instructor-help-page` for consistency, not just the snapshot in the issue.